Jenkinsfile

```diff
@@ -19,30 +18,11 @@ node {
       maxConcurrentTotal: 0,
       paramsToUseForLimit: 'release',
       throttleEnabled: true,
```

Comment:
There's no throttling on the old configuration https://ci.dev.publishing.service.gov.uk/job/govuk_release/configure